### PR TITLE
feat: change owner ui

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes #MF-3075 MF-3079 MF-3078

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

[MF-3075]: https://mask.atlassian.net/browse/MF-3075?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[MF-3079]: https://mask.atlassian.net/browse/MF-3079?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[MF-3078]: https://mask.atlassian.net/browse/MF-3078?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/mask/src/extension/popups/pages/Wallet/ChangeOwner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+import { useTitle } from '../../../hook/useTitle.js'
+
+const useStyles = makeStyles()((theme) => ({
+    content: {
+        padding: theme.spacing(2),
+    },
+    title: {
+        fontSize: 12,
+        lineHeight: '16px',
+        color: '#1C68F3',
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Wallet/ChangeOwner/index.tsx:29]: Use a theme color.

File: packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx [hunk 2]
```diff
@@ -99,8 +104,14 @@ const Logout = memo(() => {
         }
         navigate(PopupRoutes.Personas, { replace: true })
     }, [selectedPersona, history])
+
+    const manageWallets = useMemo(() => {
+        return wallets.filter((x) => x.identifier?.toText() === selectedPersona?.identifier.toText())
+    }, [wallets, selectedPersona])
```
Comment (UncleBill) [inline packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx:110]:
```suggestion
const personaId = selectedPersona?.identifier.toText()
const manageWallets = useMemo(() => {
        return wallets.filter((x) => x.identifier?.toText() === personaId)
    }, [wallets, personaId])
```

File: packages/mask/src/extension/popups/pages/Wallet/DeleteWallet/index.tsx [hunk 3]
```diff
@@ -126,6 +129,10 @@ const DeleteWallet = memo(() => {
         }
     }, [wallet, password, Wallet, connection])
 
+    const manageWallets = useMemo(() => {
+        return wallets.filter((x) => isSameAddress(x.owner, wallet?.address))
+    }, [wallets, wallet])
```
Comment (UncleBill) [inline packages/mask/src/extension/popups/pages/Wallet/DeleteWallet/index.tsx:134]:
```suggestion
    }, [wallets, wallet?.address])
```

**Review by guanbinrui — DISMISSED** [review]

File: packages/mask/src/extension/popups/pages/Wallet/ChangeOwner/index.tsx [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+    const wallets = useWallets()
+
+    const { selectedWallet: contractAccount } = useContainer(WalletContext)
+
+    const { chainId } = useChainContext<NetworkPluginID.PLUGIN_EVM>()
+    const { value: personaManagers } = useAsync(async () => {
+        return Services.Identity.queryOwnedPersonaInformation(true)
+    }, [])
+
+    console.log(personaManagers)
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Wallet/ChangeOwner/index.tsx]:
```suggestion
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**